coverage.sh

```diff
@@ -1,17 +0,0 @@
-#!/usr/bin/env nash
```

Comment:
Ow I see now that you said : I've changed the coverage to make it drop-in replacement in nash project."

But I have no idea what you mean with that :-(